specs/www.rst

```diff
@@ -236,6 +236,17 @@ Keys:
   ignored. Optional; if missing defaults to ``False``.
 
 
+Disconnected Client - ``send`` exception
+''''''''''''''''''''''''''''''''''''''''
+
+If ``send()`` is called on a closed connection the server should raise
+a server-specific subclass of ``IOError``.
+Applications may catch this exception and do cleanup work before
+re-raising it or returning with no exception.
+Servers must be prepared to catch this exception if they raised it and
+should not log it as an error in their server logs.
```

Comment:
@andrewgodwin Isn't an update to this chapter https://asgi.readthedocs.io/en/latest/specs/www.html#disconnect-receive-event-ws also necessary? I find it misleading with the actual context if someone is reading it and now knowing/understanding it derives from https://asgi.readthedocs.io/en/latest/specs/www.html#disconnected-client-send-exception

Reply:
No, they are not related to each other - the disconnect event is emitted when you call `receive()`, and the disconnected error is if you try to call `send()` once the other side has disconnected.

I'm happy to add more clarification there if you think it's still confusing.

Reply:
You're fully right, I've mixed up the things and I needed to give it a fresh look later on to catch up better on it.

I still think that if someone is looking up for infos/specs for WebSockets send events, it's unlikely that this person (if not reading the full page) will get that this could generate a IOError subclass exception.

The structure of the page does not make explicitly clear if the exception applies for both HTTP and WebSockets scope, that could be specified more.

Also (and this is more what my original thought in the first comment was), there could a short sentence/infobox/warning for every description of a send event that this could generate an exception...

Reply:
I'll go into the spec and see if I can add a bit of explicit clarification!